neuro_san/http_sidecar/http_sidecar.py

```diff
@@ -89,14 +107,17 @@ def __call__(self):
 
         app.listen(self.http_port)
         self.logger.info({}, "HTTP server is running on port %d", self.http_port)
+        self.logger.info({}, "HTTP server is shutting down after %d requests", self.requests_limit)
         # Construct initial "allowed" list of agents:
         # no metadata to use here yet.
         self.update_agents(metadata={})
         self.logger.debug({}, "Serving agents: %s", repr(self.allowed_agents.keys()))
 
         IOLoop.current().start()
+        self.logger.info({}, "Http server stopped.")
+        grpc_server.stop()
 
```

Comment:
If we get here, our http server event loop has stopped.
Time to stop gRPC service also.